services/sleeper/docker/custom/Dockerfile

```diff
@@ -0,0 +1,109 @@
+FROM nvidia/cuda:10.2-base-ubuntu18.04 as base
+#
+#  USAGE:
+#     cd services/sleeper
+#     docker build -f Dockerfile -t sleeper:prod --target production ../../
+#     docker run sleeper:prod
+#
+
+LABEL maintainer="Andrei Neagu <neagu@itis.swiss>"
+
+# simcore-user uid=8004(${SC_USER_NAME}) gid=8004(${SC_USER_NAME}) groups=8004(${SC_USER_NAME})
+ENV SC_USER_ID 8004
+ENV SC_USER_NAME scu
+RUN adduser --uid ${SC_USER_ID} --disabled-password --gecos "" --shell /bin/sh --home /home/${SC_USER_NAME} ${SC_USER_NAME}
+
+RUN apt-get update --fix-missing && \
+    apt-get install -y wget bzip2 ca-certificates curl git python3 python3-pip jq && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+
+# -------------------------- Build stage -------------------
+# Installs build/package management tools and third party dependencies
+#
+# + /build             WORKDIR
+#
+
+FROM base as build
+
+ENV SC_BUILD_TARGET build
+ENV SC_PIP pip3
+
+# ------------------------------------------------------------------------------------
+#TODO:
```

Comment:
MINOR: Please remove all comments if does not apply 
If you need a comment to remain, please add a comment explaining the reason